Source/WebKit/WebProcess/WebPage/WebPage.cpp

```diff
@@ -5323,7 +5323,7 @@ bool WebPage::findStringFromInjectedBundle(const String& target, OptionSet<FindO
 
 void WebPage::findStringMatchesFromInjectedBundle(const String& target, OptionSet<FindOptions> options)
 {
-    findController().findStringMatches(target, options, 0);
+    findController().findStringMatches(target, options, 0, { });
```

Comment:
Why not use a completion handler here which calls `send(Messages::WebPageProxy::DidFindStringMatches(string, matchRects, result.indexForSelection))`?

Then you don't need branching in `FindController`.

Reply:
That does seem better, will fix.